geomstats/geometry/euclidean.py

```diff
@@ -6,7 +6,11 @@
 
 
 class Euclidean(VectorSpace):
-    """Class for Euclidean spaces.
+    """
+    Class for Euclidean spaces.
```

Comment:
Notice this is not allowed by linting (check e.g. [here](https://github.com/geomstats/geomstats/runs/8226913786?check_suite_focus=true)).

It is expected that the first line of the docstring contains a brief description of the class/function/method.